### PR TITLE
v1.2.4

### PR DESCRIPTION
**Pull request by hsoft**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## [1.2.4](https://github.com/fgrehm/vagrant-lxc/compare/v1.2.3...v1.2.4) (Dec 12, 2017)
+
+  - Support alternative `lxcpath` [[GH-413]]
```
Comment (fgrehm) [inline CHANGELOG.md]: Sorry, do u mind creating the links as well for the issues? I just noticed that some of the ones in the other changelog entries are broken :( I fixed some of them [in master now](https://github.com/fgrehm/vagrant-lxc/commit/b96ba86f72e277accd826268583d45ee10299ad5)

**Comment by hsoft** [comment]
Hum, you're right, I hadn't noticed.

**Comment by hsoft** [comment]
there we go

**Comment by hsoft** [comment]
@fgrehm ping :)

**Comment by hsoft** [comment]
@fgrehm is there anything I can do to help?

**Comment by fgrehm** [comment]
Sorry, didn't have the time to do the travis stuff so just added you as a gem owner 🤘

**Comment by hsoft** [comment]
Alright thanks. I'll push a 1.2.4 release soon.

**Pull request merged**